### PR TITLE
Openhab USER_ID can now be set via variable at container creation time.

### PR DESCRIPTION
**Pull request by legacycode**
This is an idea from issue #55. The USER_ID can now be set at container creation time. I also choosen a high id 9001 that hopefully will not exist on the host system :-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # openHAB2 Docker Containers
+![](https://github.com/microsoft/openhab/openhab-docker/raw/master/images/openhab.png)
```
Comment (Nzbuu) [inline README.md]: Are you sure this is right?

Comment (legacycode) [inline README.md]: we will see :-)

Comment (Nzbuu) [inline README.md]: I think the URL should be something like `https://github.com/openhab/openhab-docker/raw/master/images/openhab.png`

Comment (legacycode) [inline README.md]: oh. you are right. it was a copy past error :(

**Comment by legacycode** [comment]
@Nzbuu thanks for that review 👍

File: README.md [hunk 2]
```diff
@@ -26,8 +27,8 @@ When not explicitly set, files are placed under [![Eclipse license](https://img.
 
 #### Version:
 
-* ``2.0.0`` Stable OH2 version
-* ``2.1.0-SNAPSHOT`` Current OH2 snapshot version
+* ``2.0.0`` Stable Openhab version
```
Comment (kaikreuzer) [inline README.md]: @legacycode Just as a reminder: It is spelled "openHAB", not "Openhab", so please make sure to do this right in all your work - thanks!

Comment (cniweb) [inline README.md]: Yes, Sorry, my mistake...

**Pull request merged**